src/database/index.tsx

```diff
@@ -85,6 +85,27 @@ const getDb = async () => {
   return firebase.firestore();
 };
 
+const buildCreatePayload = (entity: any) => {
+  const timestamp = currrentTimestamp();
+  return Object.assign({}, entity, {
+    createdAt: timestamp,
+    updatedAt: timestamp,
+  });
+};
+
+const buildUpdatePayload = (entity: any) => {
+  const timestamp = currrentTimestamp();
+  const payload = Object.assign({}, entity, {
+    updatedAt: timestamp,
+  });
+
+  // We don't want to actually store the id as a field. It should
+  // only be used as a CloudStore document reference.
```

Comment:
I keep seeing CloudStore references in this PR. Do we mean _Firestore_ or is this a term I'm just not familiar with in the stack?

Reply:
The full name is "Cloud Firestore", but we should just refer to it as "Firestore".  "Cloudstore" was an invention of my sleepy brain